user/how-to-guides/how-to-organize-your-qubes.md

```diff
@@ -0,0 +1,451 @@
+working on. Of course, he knows that it could just as easily be malware
+intended to infect his computer. Qubes OS is essential for Bob, since it allows
+him to handle all this suspicious data securely, keeping it compartmentalized
+so that it doesn't risk infecting the rest of his machine.
+
+Bob isn't a super technical guy. He prefers to keep his tools simple so he can
+focus on what's important to him: uncovering the truth, exposing the guilty,
+exonerating the innocent, and shining light on the dark corners of society. His
+mind doesn't naturally gravitate to the technical details of how his computer
+works, but he's aware that people are getting hacked all the time and that the
+nature of his work might make him a target. He wants to protect his sources,
+his colleagues, his family, and himself; and he understands that computer
+security is an important part of that. He has a Qubes laptop that he uses only
+for work, which contains:
+
+- One offline qube for writing. It only runs LibreOffice Writer. This is where
+  Bob does all of his writing. This window is usually open side-by-side with
+  another window containing research or material from a source.
+
+- Multiple email qubes. One is for receiving emails from the general public.
+  Another is for emailing his editor and colleagues. Both are based on a
+  [minimal template](/doc/templates/minimal/) with Thunderbird installed. He's
+  configured both to open all attachments in
+  [disposables](/doc/how-to-use-disposables/) that are offline in case an
+  attachment contains a beacon that tries to phone home.
+
+- Whonix qubes. He has the standard `sys-whonix` service qube for providing
+  Torified network access, and he uses disposable `anon-workstation` app qubes
+  for using Tor Browser to do research on stories he's writing. Since the topic
+  is often of a sensitive nature or might involve powerful individuals, it's
+  important that he be able to conduct this research with a degree of
+  anonymity. He doesn't want the subjects of his investigation to know that
+  he's investigating him. He also doesn't want his network requests being
+  traced back to his work or home IP addresses. Whonix addresses both of these
+  concerns. He also has another Whonix-based disposable template for receiving
+  tips anonymously via Tor, since some whistleblowers he's interacted with have
+  said that they don't want to risk using regular email.
+
+- Two qubes for
+  [Signal](https://github.com/Qubes-Community/Contents/blob/master/docs/privacy/signal.md).
+  Bob has two Signal app qubes (both on the same template in which the Signal
+  desktop app is installed). One is linked to his work mobile number for
+  communicating with co-workers. The other is a public number that serves as
+  another method of allowing sources to contact him confidentially. This is
+  especially useful for individuals who aren't tech-savvy enough for Tor but
```

Comment:
what do you think about "aren't tech-savvy enough for Tor or don't have convenient access to a laptop, but have a smart phone" ?

Reply:
A fair number of people actually do use Tor on their smartphones nowadays, so I think the laptop/smartphone distinction might be a bit orthogonal. However, you're right to point out that lack of tech-savviness is not the only (and perhaps not even the most likely) reason someone might not use Tor, so here's my attempted rewording to make the statement both simpler and more generally-applicable: "This is especially useful for individuals who don't use Tor but for whom unencrypted communication could be dangerous."